src/mono/mono.proj

```diff
@@ -32,9 +32,29 @@
         <_MonoExtraCFLAGS>-O2 -g</_MonoExtraCFLAGS>
         <_MonoExtraCXXFLAGS>-O2 -g</_MonoExtraCXXFLAGS>
     </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm64' ">
+        <_MonoTuple>aarch64-linux-gnu</_MonoTuple>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm' ">
+        <_MonoTuple>arm-linux-gnueabihf</_MonoTuple>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetArchitecture)' == 'arm' Or '$(TargetArchitecture)' == 'arm64' ">
```

Comment:
we should probably not set all these parameters when we're not cross compiling, e.g. if you directly build on an ARM machine